Core/src/test/java/com/plotsquared/core/plot/FlagTest.java

```diff
@@ -74,4 +77,60 @@ public void testFlagName() {
         Assertions.assertEquals("use", flagName);
     }
 
+    @Test
+    public void shouldSuccessfullyParseTitleFlagWithTitleSingularAndSubTitleEmpty() {
+        Assertions.assertDoesNotThrow(() -> {
+            var title = PlotTitleFlag.TITLE_FLAG_DEFAULT.parse("\"test\" \"\"").getValue();
+            Assertions.assertEquals("test", title.title());
+            Assertions.assertEquals("", title.subtitle());
+        }, "Should not throw a FlagParseException");
+    }
+
+    @Test
+    public void shouldSuccessfullyParseTitleFlagWithTitleMultipleWordsAndSubTitleEmpty() {
+        Assertions.assertDoesNotThrow(() -> {
+            var title = PlotTitleFlag.TITLE_FLAG_DEFAULT.parse("\"test hello test\" \"\"").getValue();
+            Assertions.assertEquals("test hello test", title.title());
+            Assertions.assertEquals("", title.subtitle());
+        }, "Should not throw a FlagParseException");
+    }
+
+    @Test
+    public void shouldSuccessfullyParseTitleFlagWithTitleMultipleWordsAndSubTitleMultipleWords() {
+        Assertions.assertDoesNotThrow(() -> {
+            var title = PlotTitleFlag.TITLE_FLAG_DEFAULT.parse("\"test hello test\" \"a very long subtitle\"").getValue();
+            Assertions.assertEquals("test hello test", title.title());
+            Assertions.assertEquals("a very long subtitle", title.subtitle());
+        }, "Should not throw a FlagParseException");
+    }
+
+    @Test
+    public void shouldSuccessfullyParseTitleFlagWithTitleEmptyAndSubTitleSingleWord() {
+        Assertions.assertDoesNotThrow(() -> {
+            var title = PlotTitleFlag.TITLE_FLAG_DEFAULT.parse("\"\" \"single\"").getValue();
+            Assertions.assertEquals(" ", title.title());
+            Assertions.assertEquals("single", title.subtitle());
+        }, "Should not throw a FlagParseException");
+    }
+
+    @Test
+    public void shouldExtractTitleWhenASingleDoubleQuoteAtEndOfTitle() {
+        Assertions.assertDoesNotThrow(() -> {
+            var plotTitle = PlotTitleFlag.TITLE_FLAG_DEFAULT.parse("title\"").getValue();
+            Assertions.assertEquals("title", plotTitle.title());
+            Assertions.assertEquals("", plotTitle.subtitle());
+        }, "Should not throw a FlagParseException");
+    }
+
+    @Test
+    public void shouldThrowFlagParseExceptionWithQuotesGreater4() {
+        var exception = Assertions.assertThrows(FlagParseException.class, () -> {
+            PlotTitleFlag.TITLE_FLAG_DEFAULT.parse("\"title\" \"subtitle\" \"more\"");
+        }, "Needs to throw a FlagParseException");
```

Comment:
From the looking alike, we can turn the statement lambda into an expression lambda.